src/devtools/client/webconsole/actions/toolbox.js

```diff
@@ -10,10 +10,16 @@ import { paintGraphicsAtTime } from "protocol/graphics";
 import { selectors } from "ui/reducers";
 
 export function highlightDomElement(grip) {
-  return ({ toolbox }) => {
+  return async ({ toolbox }) => {
+    if (grip.getPause() !== ThreadFront.currentPause) {
+      return;
+    }
+
     const highlighter = toolbox.getHighlighter();
-    if (highlighter) {
-      highlighter.highlight(grip);
+    const nodeFront = grip.getNodeFront();
+    if (highlighter && nodeFront) {
```

Comment:
when will highlighter not be available?

Reply:
I don't know, I just assumed the check was there for a reason. And it doesn't hurt, so I didn't want to take any chances.